src/components/CertificateTable.tsx

```diff
@@ -118,21 +118,21 @@ export class CertificateTable extends PaginatedLoader<ICertificateTableProps, IC
     }
 
     async componentDidMount() {
-        await super.componentDidMount();        
-
         if (this.props.selectedState === SelectedState.ForDemand && this.props.demand) {
             await this.initMatchingCertificates(this.props.demand);
         }
+
+        await super.componentDidMount();
     }
 
-    async componentDidUpdate(newProps: ICertificateTableProps) {
+    async componentDidUpdate(newProps: ICertificateTableProps, prevState: ICertificatesState) {
```

Comment:
Unnecessary declaration of prevState

Reply:
Thanks, forgot to remove that after testing.